py/src/ai/h2o/sparkling/H2OContext.py

```diff
@@ -48,6 +48,7 @@ def __h2o_connect(h2o_context):
         kwargs = {}
```

Comment:
It is definitely a good idea to have the options that make sense to be exposed curated, however, wouldn't it also make sense to provide a fallback mechanism so that users can always override/define an option even if is not exposed by SW? This could help with experimenting and problem-solving. WDYT?

Reply:
Yeah, good idea! I will work on it in a different PR. Thanks!